core/src/storage2/lazy/lazy_hmap.rs

```diff
@@ -272,6 +353,12 @@ where
         self.key.as_ref()
     }
 
+    /// Returns the length of the underlying entries.
+    #[cfg(test)]
+    pub fn len(&self) -> usize {
```

Comment:
You cannot and should not call this `len` since len normally refers to the number of items in a collection so this is highly confusing. What this actually is is the number of currently cached or persisted items.